src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultIEnumerableConstructibleConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class DefaultIEnumerableConstructibleConverter : JsonEnumerableConverter
+    {
+        // This method is invoked with reflection in CreateFromList.
+        private static IEnumerable CreateFromListInternal<TEnumerable, TElement>(IList sourceList)
+        {
+            List<TElement> list = new List<TElement>();
```

Comment:
```suggestion
            var list = new List<TElement>();
```

Reply:
Is there a benefit to making this change? Here and elsewhere

Reply:
I think the benefit is more readable code – avoiding to repeat complex data type when its evident.